plugin/Record/RecordManagerReadListViewAddress.php

```diff
@@ -139,17 +150,22 @@ public function getRowByName($name)
 	 *
 	 */
 
-	public function readFieldconfigByListviewId($listviewId)
+	public function getFieldconfigByListviewId(int $listviewId)
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlFields = "SELECT *
-			FROM {$prefix}oo_plugin_address_fieldconfig
-			WHERE `".esc_sql($this->getIdColumnMain())."` = ".esc_sql($listviewId)."
-			ORDER BY `order` ASC";
-
-		$result = $pWpDb->get_results($sqlFields, ARRAY_A);
+		$sql = $pWpDb->prepare(
+			"SELECT *
+			FROM %i
+			WHERE %i = %d
+			ORDER BY `order` ASC",
+			$prefix."oo_plugin_address_fieldconfig",
+			esc_sql($this->getIdColumnMain()),
```

Comment:
Auch hier wieder ist kein esc_sql notwendig
```suggestion
			$this->getIdColumnMain(),
```